src/async_fuse/fuse/mount.rs

```diff
@@ -166,7 +166,7 @@ async fn direct_mount(mount_point: &Path) -> anyhow::Result<RawFd> {
             "failed to get the file stat of mount point={mount_point:?}",
         ))?;
     let opts = format!(
-        "fd={},rootmode={:o},user_id={},group_id={}",
+        "fd={},rootmode={:o},uid={},gid={}",
```

Comment:
You cannot rename these, for these are mount arguments for syscall `mount`

Reply:
ok, I will update these.